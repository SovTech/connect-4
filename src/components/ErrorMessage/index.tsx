// Error message to show when things go wrong

import * as React from 'react';
import { H2, H3 } from '../HeaderText';
import { Wrapper } from './styles';

const ErrorMessage = () => {
  return (
    <Wrapper>
      <H2>Sommin went wrong ðŸ˜­</H2>
      <H3>Mayhaps refresh your browser</H3>
      <H3>Mayhaps check your interwebs connection</H3>
    </Wrapper>
  );
};

export default ErrorMessage;
