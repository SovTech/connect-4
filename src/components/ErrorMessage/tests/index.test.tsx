import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ErrorMessage from '../index';

it('renders a ErrorMessage correctly', () => {
  const tree = renderer
    .create(<ErrorMessage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
