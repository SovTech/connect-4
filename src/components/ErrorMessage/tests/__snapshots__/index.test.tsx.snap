// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a ErrorMessage correctly 1`] = `
<div
  className="sc-ifAKCX fVKAHz"
>
  <h2
    className="sc-bwzfXH ErJNR"
  >
    Sommin went wrong ðŸ˜­
  </h2>
  <h3
    className="sc-htpNat gLVzZr"
  >
    Mayhaps refresh your browser
  </h3>
  <h3
    className="sc-htpNat gLVzZr"
  >
    Mayhaps check your interwebs connection
  </h3>
</div>
`;
